app/lib/pages/settings/changelog_page.dart

```diff
@@ -168,33 +168,38 @@ class _ChangeParagraph extends StatelessWidget {
 }
 
 class UpdatePromptCard extends StatelessWidget {
+  String _getText() {
+    if (PlatformCheck.isWeb) {
+      return "Wir haben bemerkt, dass du eine veraltete Version der App verwendest. Lade die Seite neu, um die neuste Version zu erhalten! 👍";
+    }
+
+    return "Wir haben bemerkt, dass du eine veraltete Version der App installiert hast. Lade dir deswegen jetzt die Version im ${PlatformCheck.isMacOsOrIOS ? "App Store" : "Play Store"} herunter! 👍";
+  }
+
   @override
   Widget build(BuildContext context) {
     return AnnouncementCard(
       key: ValueKey('UpdatePromptCard'),
-      onTap: () => launchURL(getStoreLink()),
+      onTap: PlatformCheck.isWeb ? null : () => launchURL(getStoreLink()),
       title: "Neues Update verfügbar!",
       padding: const EdgeInsets.all(0),
       titleColor: Colors.white,
       color: const Color(0xFFF44336),
       content: Text(
-        "Wir haben bemerkt, dass du eine veraltete Version der App installiert hast. Lade dir deswegen unbedingt jetzt die Version im ${ThemePlatform.isCupertino ? "AppStore" : "PlayStore"} herunter! 👍",
+        _getText(),
         style: const TextStyle(color: Colors.white),
       ),
     );
   }
 
   String getStoreLink() {
-    // Öffnet direkt den Mac AppStore (funktioniert nur auf macOS-Geräten)
+    // Opens the Mac AppStore directly (only works on macOS devices)
     if (PlatformCheck.isMacOS) return 'https://sharezone.net/macos-direct';
 
-    // Es sollte für iOS & Android NICHT https://sharezone.net/ios oder
-    // https://sharezone.net/android verwendet werden, weil der Nutzer direkt
-    // wieder in die App geleitet wird, wenn diese installiert ist. Der Nutzer
-    // wird nur in den Store geleitet, wenn dieser die App nicht installiert hat.
-    //
-    // Der "onelink.to" leitet immer in den Store - egal, ob die App installiert
-    // ist oder nicht.
-    return "http://onelink.to/eqkmz5";
+    // It should NOT be used for iOS & Android https://sharezone.net/ios or
+    // https://sharezone.net/android because the user will be taken straight
+    // back to the app once it is installed. The user is only redirected to the
+    // store if the app is not installed. has.
+    return "http://sharezone.net/mobile-store";
```

Comment:
```suggestion
    // Instead of using https://sharezone.net/ios or
    // https://sharezone.net/android we use the link below
    // because it will always lead to the app store, regardless
    // of whether the app is already installed or not.
    // The iOS and Android links would just lead the user
    // back to this app (since the app is already installed)
    // which is not what we want since the user should tap 
    // "update" in the app store.
    return "http://sharezone.net/mobile-store";
```

Reply:
I also noticed that https://sharezone.net/android are opening the play store and not the app. Therefore, the comment is incorrect and can be deleted. This was the case https://sharezone.net/app because it uses a Firebase Dynamic Link but not for the normal https://android & ios links.